LoopKit/HealthKitSampleStore.swift

```diff
@@ -109,10 +109,11 @@ public class HealthKitSampleStore {
     ///
     /// - Parameters:
     ///   - toShare: Whether to request write authorization. Defaults to true.
+    ///   - read: Whether to request read authorization. Defaults to true.
```

Comment:
[nit] just the name `read` seems to indicate a verb about reading related to authorization.  Maybe consider `toRead` to match `toShare`?

Reply:
It matches a function with a similar signature on `HKHealthStore` (part of `HealthKit`).